### DISCUSSION TITLE
Installing and running pyamihtml

### DISCUSSION THREAD

**Discussion by petermr** [petermr/pyamihtml#3] [General]
This is for records and discussion about intalling , checking out, and running pyamihtml.
It's pafrticularly important to record all machine and environment details.

**Comment by petermr** [comment]
Requirements:
* installed Python >3.8
Installation
* git clone from pyamihtml repository

**Comment by mrchristian** [comment]
On Windows 10 Education - Powershell

Following the instructions here https://semanticclimate.github.io/p/en/posts/alpha-tests-for-fsci-23/ 

At: Test 01

pip install pyamihtml==0.0.7a1
pyamihtml IPCC --help 

after running 'pyamihtml IPCC --help ' I get the following message, is this running properly.

QUOTE

PS Microsoft.PowerShell.Core\FileSystem::\\tib.tibub.de\DFS0\Home\worthingtons\Documents\sc\pyamihtml> pyamihtml IPCC --help
pyamihtml : The term 'pyamihtml' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of 
the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ pyamihtml IPCC --help
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (pyamihtml:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
 
 UNQUOTE

**Comment by petermr** [comment]
I'm not on Windows but can suggest the following:
* list a command which *does* work such as `pygetpapers`
* ask `which pygetpapers` to find where it is installed. (don't know if
`which` is standard)
* echo your %PATH% variable
* does this information match up?
* is there a `pyamihtml` in the same library as `pygetpapers`

If there is no `pyamihtml` then it probably hasn't been installed
If there is try uninstalling and re-installing. - I have sometimes had to
do this

*ask if others can run `pyamihtml` and where it is installed.

Off for walk




On Tue, Nov 21, 2023 at 9:23 AM Simon Worthington ***@***.***>
wrote:

> On Windows 10 Education - Powershell
>
> Following the instructions here
> https://semanticclimate.github.io/p/en/posts/alpha-tests-for-fsci-23/
>
> At: Test 01
>
> pip install pyamihtml==0.0.7a1
> pyamihtml IPCC --help
>
> after running 'pyamihtml IPCC --help ' I get the following message, is
> this running properly.
>
> QUOTE
>
> PS Microsoft.PowerShell.Core\FileSystem::\tib.tibub.de\DFS0\Home\worthingtons\Documents\sc\pyamihtml>
> pyamihtml IPCC --help
> pyamihtml : The term 'pyamihtml' is not recognized as the name of a
> cmdlet, function, script file, or operable program. Check the spelling of
> the name, or if a path was included, verify that the path is correct and
> try again.
> At line:1 char:1
>
>    - pyamihtml IPCC --help
>    -
>
>      + CategoryInfo          : ObjectNotFound: (pyamihtml:String) [], CommandNotFoundException
>      + FullyQualifiedErrorId : CommandNotFoundException
>
>
>
> UNQUOTE
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/pyamihtml/discussions/3#discussioncomment-7628228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCSYRFX4GPJIJ3X5Q2ATYFRXHLAVCNFSM6AAAAAA7UEQOSWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMRYGIZDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by poulomiG99** [comment]
- Working on pyami html
- Installed Python 
- Cloned the pyamihtml from GitHub repository.
- Unable to run the test on VS Code.

**Comment by petermr** [comment]
Thanks Poulomi,
Glad you cloned the repo! Well done
"Unable to run the program on VS Code."
Please give more details .
* What did you do?
* what did you expect to happen?
* what happened? (give formatted text output, not screenshot)
* what you think the problem is
* are you blocked? Do you have further actions



On Tue, Nov 21, 2023 at 8:53 PM poulomiG99 ***@***.***> wrote:

> Working on pyami html
> Installed Python
> Cloned the pyamihtml from GitHub repository.
> Unable to run the program on VS Code.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/pyamihtml/discussions/3#discussioncomment-7635252>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCSZ4W4OWS4YWFJAVQRTYFUIFPAVCNFSM6AAAAAA7UEQOSWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMZVGI2TE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by poulomiG99** [comment]
- Installed cytoscape. 
- Got the network using cytoscape.
- Using the link.csv file from  pyamihtml-temp-unfccc-link.csv ( this is the pathway) .
